manifest.php

```diff
@@ -28,7 +28,7 @@
     'label' => 'LTI Client Diagnostic',
     'description' => 'Grants access to the client diagnostic functionality using LTI',
     'license' => 'GPL-2.0',
-    'version' => '1.1.0',
+    'version' => '1.2.0',
     'author' => 'Open Assessment Technologies SA',
     'requires' => [
         'taoLti' => '>=1.7.1',
```

Comment:
it should have a dependency to https://github.com/oat-sa/extension-tao-clientdiag/pull/130 so taoClientDiagnostic should be >= 2.3.0, right ?